pkg/controller/cainjector/controller.go

```diff
@@ -131,6 +131,7 @@ func (r *genericInjectReconciler) Reconcile(req ctrl.Request) (ctrl.Result, erro
 	if err := r.Client.Get(ctx, req.NamespacedName, target.AsObject()); err != nil {
 		if dropNotFound(err) == nil {
 			// don't requeue on deletions, which yield a non-found object
+			log.V(logf.TraceLevel).Info("not found", "err", err)
```

Comment:
Leaving this log line because it helped me to diagnose that the reconciler client was using a different cache than the event sources, so not always seeing the injectable that triggered the reconcile.